tools/push-data-update.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Do a fresh pull in case any changes have been pushed to the branch.
+git pull "https://${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git" HEAD:${GITHUB_REF}
```

Comment:
Note that this is a copy/paste of the below `push` command but changed to a `pull` instead, so I think it should be correct. 😁